compute/cker/include/cker/operation/BatchMatMul.h

```diff
@@ -99,6 +99,8 @@ class BatchMatMul
     const float *new_rhs_data = adj_y ? rhs_data : _temp_rhs.data();
 
     // Note we pass RHS args first, LHS args second
+    // Check accumulative dimensions of lhs and rhs of are equal
+    assert(Shape::ExtendedShape(5, new_rhs_shape).Dims(4) == Shape::ExtendedShape(5, new_lhs_shape).Dims(3));
```

Comment:
What is meaning of 5?

Reply:
BatchMatMul kernel supports rank up to 5